collaboration/plans/inbox/planspec.yml

```
@@ -10,3 +10,95 @@ tasks:
       - Polls every 30s
       - No console errors on preview
     changescope: [ui/**]
+plan_id: 2025-09-04-stabilize-develop-release
```

YAMLlint (1.37.1)

[error] duplication of key "plan_id" in mapping

(key-duplicates)